frontend/src/components/Entity.vue

```diff
@@ -38,19 +38,19 @@ export default {
       tabs: [
         {
           name: 'Show/Edit',
-          component: shallowRef(EntityForm),
+          component: markRaw(EntityForm),
```

Comment:
As for why markRaw: I feel that it's more appropriate, since we don't any reactivity **at all**  when passing a Vue component object. It shouldn't make any difference, but even the docs suggest that in the [markRaw](https://vuejs.org/api/reactivity-advanced.html#markraw)

...and yes, you're right I should probably change all the occurrences.